administration/src/util/translations/de.json

```diff
@@ -628,7 +628,8 @@
     "applicationToCsvError": "Es ist ein Fehler beim Export des Antrags aufgetreten.",
     "freinetPersonDataInvalid": "Erforderliche Personendaten von Freinet sind nicht verfügbar. Die Kartendaten konnten deshalb nicht übertragen werden.",
     "freinetCardDataInvalid": "Die Kartendaten können nicht an Freinet übertragen werden, weil diese ungültig sind.",
-    "freinetDataTransferNotPermitted": "Für diese Region ist keine Datenübermittlung erlaubt."
+    "freinetDataTransferNotPermitted": "Für diese Region ist keine Datenübermittlung erlaubt.",
```

Comment:
❓ The wording here is maybe a bit misleading and harsh. It should probably not be "erlaubt" but "eingerichtet" or "vorgesehen".